Misc improvements (#103)
sharetribe/flex-docs: src/docs/background/01-overview/index.md

### The Marketplace API

The Marketplace API is how your UI connects to the Flex services. It's an HTTP

> interface that by design draws influences from the [JSON API

Reviewer: I think something like this could be a bit more clear:

```suggestion
interface with a design influenced by the [JSON API
```
